packages/theme-chalk/src/backtop.scss

```
@@ -26,4 +26,30 @@
+      linear-gradient(
+        var(--el-backtop-progress-bg-color),
+        var(--el-backtop-progress-bg-color)
+      ),
+      conic-gradient(
+        getCssVar('backtop', 'text-color') var(--el-backtop-progress, 0turn),
+        getCssVar('border-color-light') 0
+      );
+    background-origin: border-box;
+    background-clip: padding-box, border-box;
+
+    &:hover {
```

Stylelint (17.14.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)